modules/aws-emr-sgs/security-group-rules.tf

```diff
@@ -539,6 +539,30 @@ resource "aws_security_group_rule" "region_server_port_add_core_sg_tamr_sgs" {
   description              = "Region Server - EMR Dashboard"
 }
 
+//Region Server rule - Additional SG for EMR Core with TAMR CIDRS for accessing HBase Metrics port
+resource "aws_security_group_rule" "region_server_port_add_core_sg_tamr_cidrs" {
+  count             = local.tamr_cidrs_provided ? 1 : 0
+  from_port         = 16030
+  to_port           = 16030
+  protocol          = "tcp"
+  security_group_id = aws_security_group.emr_additional_core.id
+  type              = "ingress"
+  cidr_blocks       = var.tamr_cidrs
+  description       = "Region Server - HBase Metrics"
+}
+
+//Region Server rule - Additional SG for EMR Core with TAMR SGs for accessing HBase Metrics port
+resource "aws_security_group_rule" "region_server_port_add_core_sg_tamr_sgs" {
```

Comment:
Name is a duplicate from above.
https://github.com/Datatamer/terraform-aws-emr/pull/45/files#diff-31f8037b12d08f6f61a61459bde4dbda5aa8125ace0c1728a10c51c332d4ea2bR531

Reply:
Oops, good catch. Fixed.